docs/how-to-guides/feathr-input-format.md

```diff
@@ -20,8 +20,29 @@ Many Spark users will use delta lake format to store the results. In those cases
 Please note that although the results are shown as "parquet", you should use the path of the parent folder and use `delta` format to read the folder.
 
 # TimePartitionPattern for input files
-When data sources are defined by 'HdfsSource', feathr supports 'time_partition_pattern' to match paths of input files. For example, given time_partition_pattern = 'yyyy/MM/dd' and a 'base_path', all available input files under paths 'base_path'/yyyy/MM/dd will be visited and used as data sources.
+When data sources are defined by `HdfsSource`, feathr supports `time_partition_pattern` to match paths of input data source files. For example, given time_partition_pattern = 'yyyy/MM/dd' and a 'base_path', all available input files under paths '{base_path}/{yyyy}/{MM}/{dd}' may will be visited and used as data sources.
+
+This pattern of path will be treated as 'timestamp' of the related data for both feature join and feature generation. E.g If the path is '{base_path}/2020/05/20', timestamp of this piece of data would be treated as '2020-05-20'
+
+This pattern can only be worked with aggregation features for now. It cannot be recognized for other cases.
+
+## How to control paths to visit
+Normally, it's not necessary to visit all data sources that match the path pattern. We may only need parts of them to be used in our jobs. Feathr have different ways to support that for feature generation and feature join. 
```

Comment:
High level comments - we should try to use user facing APIs, rather than `feature join` or `feature gen` which is unknown to users